src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write your code here
+const elements = [...document.querySelectorAll('.population')];
+
+const values = elements.map((element) => {
+  const number = parseInt(element.textContent.replace(/,/g, ''), 10);
+
+  if (isNaN(number)) {
+    return;
+  }
+
+  return number;
+});
+
+const total = values.reduce((sum, value) => sum + value, 0);
```

Comment:
If `values` contains `undefined` due to the previous mapping, the `reduce` function will result in `NaN` for `total`. Ensure that `values` only contains numbers.